integration/monai/README.md

```diff
@@ -0,0 +1,38 @@
+# MONAI Integration
+
+## Objective
+Integration with [MONAI](https://monai.io/)'s federated learning capabilities.
+
+Add `Executor` class to allow using [MONAI bundle](https://docs.monai.io/en/latest/bundle.html) configuration files with MONAI's `ClientAlgo` class.
+
+### Goals:
+
+Allow the use of bundles from the MONAI [model zoo](https://github.com/Project-MONAI/model-zoo) or custom configurations with NVFlare.
+
+### Non-goals:
+
+n/a
+
+## Background
+MONAI allows the definition of AI models using the "bundle" concept. 
+It allows for easy experimentation and sharing of models that have been developed using MONAI.
+Using the bundle configurations, we can use MONAI's `MonaiAlgo` class to execute a bundle model in a federated scenario using NVFlare.
+
+## Description
+NVFlare executes the `MonaiAlgo` class using the `ClientAlgoExecutor` class provided with this package.
```

Comment:
`MonaiAlgo` should it be changed to MONAI's `ClientAlgo` class? 
Because from the codes here I only see the ClientAlgo class.

People that interested in `ClientAlgo` will go to MONAI documentation and found how that is written.

Reply:
Thank you. I changed the readme to make this more clear.